### PR TITLE
iOS/update getting-started guide, removed refs to amplify-app

### PR DESCRIPTION
**Pull request by diegocstn**
*Description of changes:*
Remove references of `amplify-app` from iOS tutorial and getting started guide:

#### Getting started
**Before:** https://docs.amplify.aws/lib/datastore/getting-started/q/platform/ios
**After:**
data-store-getting-started

#### Tutorial
**Before**:  https://docs.amplify.aws/start/getting-started/setup/q/integration/ios

**After:**
tutorial

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by drochetti — COMMENTED** [review]
lgtm!

**Review by drochetti — APPROVED** [review]

File: docs/lib/datastore/fragments/ios/getting-started/30_setupEnv.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<inline-fragment src="~/lib/datastore/fragments/native_common/setup-env-cli.md"></inline-fragment>
+
+Once the command is done, a new group named `AmplifyConfig` should be added to your project and contain the following files:
```
Comment (jamesonwilliams) [inline docs/lib/datastore/fragments/ios/getting-started/30_setupEnv.md:3]:
```suggestion
The `amplify add api` command adds a new `AmplifyConfig` group to your Xcode project. It will contain the following files:
```

File: docs/lib/datastore/fragments/ios/getting-started/40_codegen.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
-On iOS the aforementioned `amplify-app` offers an Xcode integration that automatically adds the Amplify-specific files to your project.
+On iOS the `amplify` CLI offers an Xcode integration that automatically adds the Amplify-specific files to your project.
```
Comment (jamesonwilliams) [inline docs/lib/datastore/fragments/ios/getting-started/40_codegen.md:1]:
```suggestion
The `amplify` CLI is integrated with Xcode and will automatically add Amplify-specific files to your project.
```

File: docs/lib/datastore/fragments/ios/getting-started/40_codegen.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 
 1. **Run the command**:
     ```
-    amplify codegen models && amplify-app
+    amplify codegen models
     ```
 
-1. Once the command finishes, a new group named `AmplifyModels` should be added to your project and contain the following files:
+2. Once the command finishes, a new group named `AmplifyModels` should be added to your project and contain the following files:
```
Comment (jamesonwilliams) [inline docs/lib/datastore/fragments/ios/getting-started/40_codegen.md:8]:
```suggestion
2. The `amplify codegen models` command adds a new `AmplifyModels` group to your Xcode project. It will contain the following files:
```

File: docs/lib/datastore/fragments/native_common/setup-env-cli.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+If you don't have an existing project already, create a new Amplify project by running:
```
Comment (jamesonwilliams) [inline docs/lib/datastore/fragments/native_common/setup-env-cli.md:1]:
```suggestion
To use Amplify, you must first initialize it for use in your project. If you haven't already done so, run this command:
```

File: docs/lib/datastore/fragments/native_common/setup-env-cli.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    `Auto Merge`
+? Do you have an annotated GraphQL schema?
+    `No`
+? Choose a schema template
+    `Single object with fields (e.g., “Todo” with ID, name, description)`
+```
+
+<amplify-callout warning>
+
+**Troubleshooting:** Cloud sync will fail without the **conflict detection** configuration. In that case use `amplify update api` and choose **Enable DataStore for entire API** (this option will enable the conflict detection as described above).
```
Comment (jamesonwilliams) [inline docs/lib/datastore/fragments/native_common/setup-env-cli.md:45]:
```suggestion
**Troubleshooting:** Cloud sync will fail without the **conflict detection** configuration. To enable it for an existing project, run `amplify update api` and choose **Enable DataStore for entire API**.
```

File: docs/lib/datastore/fragments/native_common/setup-env-cli.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+```console
+# For new APIs
+amplify add api
+
+# For existing APIs
+amplify update api
+```
+
+During the API configuration process select **GraphQL** as the API type and reply to the questions as follows. Make sure you respond **Yes, I want to make some additional changes** when prompted for **advanced settings** and turn on **conflict detection**. This setting is **required** when syncing data to the cloud since the conflict resolution strategy is what allows local data to be reconciled with data from the cloud backend.
```
Comment (jamesonwilliams) [inline docs/lib/datastore/fragments/native_common/setup-env-cli.md:16]:
```suggestion
The CLI will prompt you to configure your API. Select **GraphQL** as the API type and reply to the questions as shown below. When prompted to configure **advanced settings**, respond **Yes, I want to make some additional changes**, and ensure that **conflict detection** is enabled. Conflict detection is **required** when using the DataStore to sync data with the cloud.
```

**Review by jamesonwilliams — APPROVED** [review]
Looks good. Tried to simplify the wording in a few places.

File: docs/lib/datastore/fragments/ios/getting-started/40_codegen.md (hunk 2 above)
Comment (diegocstn) [reply docs/lib/datastore/fragments/ios/getting-started/40_codegen.md:1]: I wouldn't change the wording here as unfortunately the CLI is not exactly "integrated" with Xcode (at least not as integrated as it currently is with Android Studio) :)
